opsdroid/connector.py

```diff
@@ -72,6 +72,15 @@ def __init__(self, config):
         """
         raise NotImplementedError
 
+    async def user_typing(self, opsdroid):
```

Comment:
I would be tempted to add an argument to pass in `True` or `False` so it can be turned on and off.

This should also `pass` instead of raising an error as it is optional functionality for a connector.